CommonData/column.cpp

```diff
@@ -1661,12 +1666,29 @@ bool Column::setStringValueToRowIfItFits(size_t row, const std::string & value,
                                 ColumnUtils::convertValueToDoubleForImport(value,				dbl))
 							typeChanged = changeType(_preEditType) == columnTypeChangeResult::changed; //Just try it
 					}
+					else if(_type == columnType::nominalText)
```

Comment:
Ah I meant to comment here...
```suggestion
					else if(_type == columnType::nominalText && (_prevEditType == columnType::nominal || _prevEditType == columnType::ordinal))
```
Like for scale?